CONTRIBUTING.md

```diff
@@ -116,9 +116,11 @@ You can run the unit tests by running:
 
 ```bash
 source .venv/bin/activate
-./scripts/unit-tests.sh
+./scripts/unit-tests.sh [TESTDIR]
```

Comment:
Instead of TESTDIR can we say that the script accepts any additional pytest arguments? TESTDIR being once of them? Thanks!

Reply:
done